api_reference.md

```diff
@@ -605,6 +626,6 @@ result.class # => Midtrans::Result
 * `Midtrans::Result#status_code` - `integer`, e.g. 200, 402. Documentation https://api-docs.midtrans.com/#status-code
```

Comment:
Why is this section under level 4 heading? it should be level 2 heading for proper section leveling.